packager/media/base/stream_info.h

```diff
@@ -41,6 +41,7 @@ enum Codec {
   kCodecAAC = kCodecAudio,
   kCodecAC3,
   kCodecAC4,
+  kCodecALAC,
```

Comment:
I was unsure, so I double-checked.  This is outside the shared library interface, so changing the numbers here won't break users of the library.  It also doesn't impact the format of --output_media_info.